src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -1110,6 +1111,12 @@ void CouplingSchemeConfiguration::setParallelAcceleration(
       checkIfDataIsExchanged(dataID);
     }
     scheme->setAcceleration(_accelerationConfig->getAcceleration());
+
+    if (dynamic_cast<acceleration::AitkenAcceleration *>(_accelerationConfig->getAcceleration().get()) != nullptr)
+      PRECICE_WARN("You configured participant \"{}\" in a parallel-implicit coupling scheme with \"Aitken\" "
+                   "acceleration, which is known to perform bad in parallel coupling schemes. Consider switching to a serial-implicit coupling "
+                   "scheme or changing the acceleration method.",
+                   participant);
```

Comment:
```suggestion
      PRECICE_WARN("You configured participant \"{}\" in a parallel-implicit coupling scheme with \"Aitken\" "
                   "acceleration, which is known to perform bad in parallel coupling schemes. "
                   "See https://precice.org/configuration-acceleration.html#dynamic-aitken-under-relaxation for details."
                   "Consider switching to a serial-implicit coupling scheme or changing the acceleration method.", participant);
```

I'm currently working on the documentation in https://github.com/precice/precice.github.io/pull/71. We could link to this place in the warning.